src/hb/ot_layout_gsubgpos.rs

```diff
@@ -273,7 +286,19 @@ impl<'a, 'b> skipping_iterator_t<'a, 'b> {
 
     pub fn prev(&mut self, unsafe_from: Option<&mut usize>) -> bool {
         assert!(self.num_items > 0);
-        while self.buf_idx >= usize::from(self.num_items) {
+
+        let mut stop = self.num_items - 1;
+
+        if self
+            .ctx
+            .buffer
+            .flags
+            .contains(BufferFlags::PRODUCE_UNSAFE_TO_CONCAT)
+        {
+            stop = 1 - 1;
```

Comment:
Isn't this just zero? Seems like it was fixed in harfbuzz later. Don't see it in master anymore.

Reply:
It is... But that's how it was in the commit 😆. https://github.com/harfbuzz/harfbuzz/commit/1fa64c0c2

But I can obviously change it.

Reply:
Yeah, I really wrote a 0 that way. There was an unimportant reason for it. :))